<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogvideo</class>
 <widget class="QDialog" name="Dialogvideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton_seek_barckward">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seek_foward">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play_pause">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_duration">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>601</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_volume">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_volume">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_video">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>701</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
border:1px solid black
}</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_time">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ouvrir">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
